ci/links.sh

```diff
@@ -1,10 +1,15 @@
 #!/usr/bin/env bash
 
+set -e
+
+FAILURE=0
+
+git config --global --add safe.directory /usr/src/app
 npm install -g markdown-link-check
 git fetch origin main:main
 # To run this on the entire repo, replace the following command with `$(find ./ -type f | grep .md)`
-for file_name in $(git diff --name-only $HEAD main); do
-  if [[ $file_name == *".md" ]]; then
-    npx markdown-link-check --config ./ci/link-config.json --progress --verbose "$file_name"
-  fi
+for file_name in $(git diff --name-only $HEAD main -- ./**/*.md ./*.md); do
```

Comment:
Same request as spelling for linting as well.

Reply:
See the first comment, it has now been fixed in the script.